pkg/config/rpc_config.go

```diff
@@ -20,6 +20,7 @@ type (
 		MaxRequestHeaderBytes     int           `yaml:"MaxRequestHeaderBytes"`
 		MaxWebSocketClients       int           `yaml:"MaxWebSocketClients"`
 		SessionEnabled            bool          `yaml:"SessionEnabled"`
+		SessionExpansionEnabled   bool          `yaml:"SessionExpansionExtension"`
```

Comment:
Don't forget to update https://github.com/nspcc-dev/neo-go/blob/master/docs/node-configuration.md#rpc-configuration and RPC extensions section.

Reply:
Not resolved.